subsys/zigbee/cli/zigbee_cli_cmd_attr_report.c

```diff
@@ -433,15 +400,15 @@ int cmd_zb_subscribe_on(const struct shell *shell, size_t argc, char **argv)
 {
 	int ret_val;
 	zb_bufid_t bufid;
-	struct configure_reporting_req_data *req_data;
+	struct configure_reporting_req *req_data;
 	struct ctx_entry *entry = ctx_mgr_new_entry(CTX_MGR_CFG_REPORT_REQ_ENTRY_TYPE);
 
 	if (entry == NULL) {
 		zb_cli_print_error(shell, "Request pool empty - wait a bit", ZB_FALSE);
```

Comment:
I think a good idea would be to inform the user for what he has to wait for. `"Request pool empty - wait until currently processed reporting request is responded or times out"`.

Reply:
I think that `Request pool empty - wait a bit` is in not incorrect - context manager is used for both ZDO & ZCL commands so any ongoing ZCL/ZDO request/command will free a ctx mgr entry when finished.